service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/PayonePostServiceImpl.java

```diff
@@ -71,6 +71,7 @@
     private static final String ENCODING_UTF8 = "UTF-8";
 
     private String serverAPIURL;
+    private static final Map<String,String> KEY_MAPPING = Collections.singletonMap("paymentData", "add_paydata[action]");
```

Comment:
Why is it here and not in a Klarna specific class?

Reply:
because it is a global mapping